DevEdu.API/DevEdu.API/Configuration/ExceptionResponses/ValidationExceptionResponse.cs

```diff
@@ -1,6 +1,38 @@
-﻿namespace DevEdu.API.Configuration
+﻿using DevEdu.Business.Exceptions;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+using System;
+using System.Collections.Generic;
+
+namespace DevEdu.API.Configuration
 {
     public class ValidationExceptionResponse
     {
+        public const int CODE = 1001;
+        public const string MESSAGE = "Validation Error";
+        public List<ValidationError> ValidationErrors { get; set; }
+
+        public ValidationExceptionResponse(Exception exception)
+        {
+            ValidationErrors = new List<ValidationError>();
+            ValidationErrors.Add(new ValidationError
+            {
+                Code = 422,
+                Message = exception.Message,
+                Description = exception.Message

```

Comment:
this part will be changed
we will have Code, Field and Message here
so you' ve got to provide field name in this case
for this purpose, you can extend ValidationException with additional property